Jenkinsfile

```diff
@@ -46,11 +46,14 @@ pipeline {
                 sh  '''
                     # Clone this repository & Parabank repository into the workspace
                     mkdir parabank-jenkins
-                    git clone https://github.com/whaaker/parabank-jenkins.git parabank-jenkins
+                    git clone -b Pawel-Selenic https://github.com/whaaker/parabank-jenkins.git parabank-jenkins
 
                     mkdir parabank
                     git clone https://github.com/parasoft/parabank parabank
 
+                    mkdir parabank-selenic
+                    git clone -b jenkins-selenic-tests https://github.com/pteodor/parabank parabank-selenic
```

Comment:
Please copy your Selenium test project files into this repo instead of cloning a separate repo from your github account.  Make a directory called 'selenic' on the same level as 'parabank-docker', 'soatest', and 'jtest' to contain a Selenium Maven project.